docs/quickstart/hello-world-precompile-tutorial.md

```diff
@@ -0,0 +1,926 @@
+
+### Prerequisites
+
+- Clone the [Subnet-EVM](https://github.com/ava-labs/subnet-evm) repo.
+- Clone [Avalanchego](https://github.com/ava-labs/avalanchego) repo.
+- Install [Avalanche Network Runner](https://docs.avax.network/subnets/network-runner)
+- Download [solc](https://docs.soliditylang.org/en/v0.8.9/installing-solidity.html)
+
+## Tutorial
+
+We will first start off by creating the Solidity interface that we want our precompile to implement. This will be the HelloWorld Interface. It will have two simple functions, `sayHello` and `setGreeting`. These two functions will demonstrate the getting and setting respectively of a value using state access. For the rest of the tutorial, we will be working in a new branch in Subnet-EVM.
+
+We will place the interface in `./contract-examples/contracts`
+
+```sol
+// (c) 2022-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+interface IHelloWorld {
+ // sayHello returns the string located at [key]
+  function sayHello() external returns (string calldata result);
+
+// setGreeting sets the string located at [key]
+  function setGreeting(string calldata response) external;
+}
+```
+
+Now we have an interface that our precompile can implement!
+Let's create an [abi](https://docs.soliditylang.org/en/v0.8.13/abi-spec.html#:~:text=Contract%20ABI%20Specification-,Basic%20Design,as%20described%20in%20this%20specification.) of our solidity code.
+
+In the same `./contract-examples/contracts` directory, let's run
+
+```
+solc --abi IHelloWorld.sol -o .
+```
+
+This spits out the abi code!
+
+IHelloWorld.abi
+
+```json
+[
+  {
+    "inputs": [],
+    "name": "sayHello",
+    "outputs": [
+      { "internalType": "string", "name": "result", "type": "string" }
+    ],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      { "internalType": "string", "name": "recipient", "type": "string" }
+    ],
+    "name": "setGreeting",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  }
+]
+```
+
+Note: The abi must have named outputs in order to generate the precompile template.
+
+### Generating the Precompile
+
+Now that we have an abi for the precompile gen tool to interact with. We can run the following command to generate our HelloWorld precompile!
+
+In the root of the repo run
```

Comment:
What repo? Did we ever specify which exact project to check out before the start of this tutorial?

Reply:
https://github.com/ava-labs/avalanche-docs/pull/1040/files#diff-1b7eb966ffdb6b059430567803d20daad29d07b34323172f94029617e7e99e6fR141

Reply:
Defined here.

Reply:
> https://github.com/ava-labs/avalanche-docs/pull/1040/files#diff-1b7eb966ffdb6b059430567803d20daad29d07b34323172f94029617e7e99e6fR141

This link directs me to https://github.com/ava-labs/avalanche-docs/pull/1040/files#

Reply:
Fixed